build/release.sh

```diff
@@ -44,6 +46,10 @@ docker_tag="google/cadvisor:$VERSION"
 gcr_tag="gcr.io/cadvisor/cadvisor:$VERSION"
 docker build -t $docker_tag -t $gcr_tag -f deploy/Dockerfile .
 
+# Build the release binary without libpfm4 to not require libpfm4 in runtime environment
```

Comment:
Is this only required if perf is enabled, or will cadvisor not run at all if libpfm4 isn't enabled?  This would also mean perf support is only available in the containerized release version of cAdvisor .

Reply:
If binary is built with libpfm4 ( with GO_FLAGS="-tags=libpfm,netgo") then cAdvisor does not run without libpfm4 in runtime environment and this mean that perf support will be available only in the containerized release version of cAdvisor .

Reply:
May release contain two binaries? The first without perf support as it is now and the second with perf support (and later with ipmctl)?

Reply:
I'm fine with what you have for this PR.  I don't want to set the precedence that we will release additional binaries for optional features.